decidim-core/config/locales/en.yml

```diff
@@ -1053,6 +1053,8 @@ en:
       show:
         administrators: Administrators
         allow_public_contact: Allow anyone to send me a direct message, even if I don't follow them.
+        component_notification_settings:
```

Comment:
Maybe this key should be moved to decidim-meetings ?

Reply:
Because this is used in decidim-core/app/views/decidim/notifications_settings/show.html.erb and, and in the future, "component_notification_settings" can have multiple values, I think that it will be better to keep it here